frontend/src/app/supportAdmin/PendingOrganizations/utils.tsx

```diff
@@ -0,0 +1,26 @@
+import * as yup from "yup";
+
+import { phoneNumberIsValid } from "../../patients/personSchema";
+
+export interface PendingOrganizationFormValues {
+  name: string;
+  adminFirstName: string | undefined;
+  adminLastName: string | undefined;
+  adminEmail: string | undefined;
+  adminPhone: string | undefined;
+}
+
+const phoneNumberIsValidOrEmpty = (input: any) =>
+  input === "" || phoneNumberIsValid(input);
+
+export const pendingOrganizationSchema: yup.SchemaOf<PendingOrganizationFormValues> = yup
+  .object()
+  .shape({
+    name: yup.string().required("Organization name is required"),
```

Comment:
org name isn't actually required for the mutation, so I don't think we need to validate on this one. They only have to enter the name if they want to change it